quickwit-doc-mapper/src/default_doc_mapper/mod.rs

```diff
@@ -29,20 +30,74 @@ pub use self::field_mapping_entry::{DocParsingError, FieldMappingEntry};
 pub use self::field_mapping_type::FieldMappingType;
 
 /// Regular expression representing the restriction on a valid field name.
-pub const FIELD_MAPPING_NAME_PATTERN: &str = r#"^[_a-zA-Z][_\.\-a-zA-Z0-9]*$"#;
+pub const FIELD_MAPPING_NAME_PATTERN: &str = r#"^[_a-zA-Z][_\.\-a-zA-Z0-9]{0,254}$"#;
 
 /// Validator for a potential `field_mapping_name`.
 /// Returns true if the name can be use for a field mapping name.
 ///
 /// A field mapping name must start by a letter `[a-zA-Z]`.
 /// The other characters can be any alphanumic character `[a-ZA-Z0-9]` or `_`, `.`, `-`.
```

Comment:
```
/// A field mapping name must start by a letter `[a-zA-Z]`.
/// The other characters can be any alphanumic character `[a-ZA-Z0-9]` or `_`, `.`, `-`.
/// It can contains at most 254 characters.
```